gbus/worker.go

```diff
@@ -194,6 +194,7 @@ func (worker *worker) reject(requeue bool, delivery amqp.Delivery) error {
 		worker.log().WithError(err).Error("could not reject the message")
 		worker.span.LogFields(slog.Error(err))
 	}
+	metrics.ReportRejectedMessage()
```

Comment:
@adiweiss I am not sure but I think this metric should be reported only if the requeue parameter is false WDYT ?